src/renderers/shared/stack/reconciler/ReactMultiChild.js

```diff
@@ -234,12 +229,13 @@ var ReactMultiChild = {
             transaction,
             this,
             this._hostContainerInfo,
-            context
+            context,
+            selfDebugID
           );
           return nextChildren;
         }
       }
-      nextChildren = flattenChildren(nextNestedChildrenElements);
+      nextChildren = flattenChildren(nextNestedChildrenElements, selfDebugID);
```

Comment:
Interesting that we weren’t passing it before (this code path could still run in `__DEV__`).